### ISSUE TITLE
./fluxion.sh -i

### ISSUE THREAD

**Issue by BedoM** [FluxionNetwork/fluxion#800]
Hello
I have Fedora 31 and the script ./fluxion.sh -i dont install all. He blocked on mdk3.

Do you have an solutions ?

**Comment by cjb900** [comment]
Fluxion is mostly designed for Kali Linux and similar Debian hacking Linux distributions. So the way it installs stuff may not work in Fedora. You would probably need to install mdk3 manually and see if that works. But I don't believe fluxion has been tested in Fedora so I can't vouch for it it will work or not. Also the developers have all but mostly abandoned this project. So they probably won't fix anything to make is work in Fedora either. If you can't get it to work you would be better off using a dedicated Linux hacking OS like Kali Linux or Parrot OS

**Comment by strasharo** [comment]
Install mdk3 manually from source, it's not in the default repositories.
